src/contracts/test/GPv2SettlementTestInterface.sol

```diff
@@ -1,10 +1,19 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
 pragma solidity ^0.6.12;
-pragma experimental ABIEncoderV2;
 
 import "../GPv2Settlement.sol";
 
 contract GPv2SettlementTestInterface is GPv2Settlement {
+    constructor(GPv2AccessControl _controller)
```

Comment:
nit: 
```suggestion
    constructor(GPv2Authentication controller_)
```

Trailing underscore is the standard in solidity: https://docs.soliditylang.org/en/v0.7.4/style-guide.html#avoiding-naming-collisions

Reply:
But I don't wanna. Do we have to?

Reply:
We should probably follow the suggested Solidity style guide IMO so that we are consistent with other Solidity code as well.

Reply:
Just to clarify, I'm not particularly attached to this particular style (in fact, I find trailing underscored aesthetically unpleasing). In general though, I think style guides are a super useful way to keep code looking consistent.